llvm/include/llvm/SandboxIR/Tracker.h

```diff
@@ -135,6 +136,26 @@ class EraseFromParent : public IRChangeBase {
 #endif
 };
 
+class RemoveFromParent : public IRChangeBase {
+  /// The instruction that is about to get removed.
+  Instruction *RemovedI = nullptr;
+  /// This is either the next instr, or the parent BB if at the end of the BB.
+  PointerUnion<Instruction *, BasicBlock *> NextInstrOrBB;
```

Comment:
maybe a helper typedef and function would be useful if this is used in a lot of `IRChangeBase`s

Reply:
Yeah I could do that in a follow-up NFC